Cargo.toml

```diff
@@ -4,6 +4,22 @@ cargo-features = []
 resolver = "2"
 members = ["cli", "lib", "lib/testutils", "lib/gen-protos"]
 
+[workspace.package]
+version = "0.8.0"
+license = "Apache-2.0"
+rust-version = "1.71"
+edition = "2021"
+readme = "README.md"
+homepage = "https://github.com/martinvonz/jj"
+repository = "https://github.com/martinvonz/jj"
+documentation = "https://github.com/martinvonz/jj"
+categories = ["version-control", "development-tools"]
+keywords = ["VCS", "DVCS", "SCM", "Git", "Mercurial"]
+authors = [
+    "Martin von Zweigbergk <martinvonz@google.com>",
+    "The Jujutsu Developers",
+]
```

Comment:
Some of your recent refactoring has reminded me of this `authors` field and I meant to ask if people thought we should put something else than my name here. I'm fine with your update, but also feel free to not list me explicitly (it does seem a bit redundant).

Alternatively, we could just delete the field altogether. I'm not sure that it's useful anyway. Some large crates like `clap` don't have it.

Reply:
Yes, it's a bit redundant if it's only "The Jujutsu Developers" for that field, so maybe it should be deleted.

FWIW, normally that kind of authorship line is what you'd see in a copyright statement somewhere, e.g. in `SPDX-FileCopyrightText: © 2019-2023 The Jujutsu Developers` or whatever, so that's where I took it from.